### PR TITLE
Upgrade to CKE v19

### PR DESCRIPTION
**Pull request by ejwill04**
## Summary

This upgrades our CKE packages to their latest versions.  Please see this link for details on the upgrade: https://github.com/ckeditor/ckeditor5/releases

## What changed in the code?

* upgraded packages
* changed naming of `snippet` to `snippets` in the plugins.  Not sure why that changed @dunncl15 

#### Gaps in Coverage (i.e. missing tests)

Manually tested the upgrade

## Anything else before merging (i.e. why is this on hold)?

No

### CODE REVIEW

File: src/Ckeditor/plugins/index.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ ClassicEditor.defaultConfig = {
       '|',
       'removeFormat',
       '|',
-      'snippet',
+      'snippets',
```
Comment (ejwill04) [inline src/Ckeditor/plugins/index.js:160]: @dunncl15 - any idea why this needed to change to the plural form to work?

Comment (dunncl15) [reply src/Ckeditor/plugins/index.js:160]: @ejwill04 I'm not sure exactly. It's always been singular (for no particular reason) and the label for the UI component was "Snippets". Was it breaking?

Comment (ejwill04) [reply src/Ckeditor/plugins/index.js:160]: Yeah, it wouldn't render the UI component and it said it couldn't find a toolbar item with that name.  Maybe it's a change they made somewhere in the upgrade.

Comment (dunncl15) [reply src/Ckeditor/plugins/index.js:160]: Hmm interesting. I wonder if the name used when adding it to the `componentLibrary` has to match that toolbar config. I had used `snippet` in the toolbar config, but `snippets` as the name when adding the ui component. Regardless I think your update makes sense.

Comment (giarcjc) [reply src/Ckeditor/plugins/index.js]: Marking as resolved since snippets have been removed from the toolbar in master branch for the time being.

**Pull request merged**